trigger/payloads.go

```diff
@@ -432,6 +428,22 @@ func makeCommonPayload(sub *event.Subscription, selectors []event.Selector,
 	return nil, errors.Errorf("unknown type: '%s'", sub.Subscriber.Type)
 }
 
+func getFailedTestsFromTemplate(t *task.Task) ([]task.TestResult, error) {
+	settings, err := evergreen.GetConfig()
+	if err != nil {
+		return nil, errors.Wrap(err, "problem getting evergreen config")
+	}
+
+	result := []task.TestResult{}
+	for i := range t.LocalTestResults {
+		if t.LocalTestResults[i].Status == evergreen.TestFailedStatus {
+			result = append(result, t.LocalTestResults[i])
+			result[len(result)-1].URL = settings.Ui.Url + t.LocalTestResults[i].URL
```

Comment:
This works but is pretty janky. Can you save t.LocalTestResults[i] as a local variable, change the URL on that, then append that local variable to result?